src/components/Chart/AreaChart.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import C3Chart from 'react-c3js';
+import ClassNames from 'classnames';
+
+require('patternfly/dist/js/patternfly-settings');
+require('patternfly/dist/js/patternfly-settings-charts');
+
+const c3ChartDefaults = window.patternfly.c3ChartDefaults();
+
+/**
+ * AreaChart Component for Patternfly React
+ */
+class AreaChart extends React.Component {
+  render() {
+    this.props.data.type = this.props.data.type || 'area';
+
+    let className = ClassNames(this.props.className, { 'area-chart-pf': true });
```

Comment:
any reason to use let vs const? (and any idea why linting doesn't complain?)

Reply:
> any idea why linting doesn't complain?

no :-) , fixed